src/adapters/postgres/cohortMembers-adapter.ts

```
@@ -1278,4 +1284,291 @@ export class PostgresCohortMembersService {
+      const form = await this.formsService.getFormData({
+        context: 'COHORTMEMBER',
+        contextId,
+        contextType: 'COHORTMEMBER',
+        tenantId,
+      });
+
+      // Step 3: For each userId, fetch their submissions and enrich
+      const enrichedResults = await Promise.all(
+        userDetails.map(async (user) => {
+          let formInfo = null;
+          if (form && form.formid && form.status === 'active') {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)